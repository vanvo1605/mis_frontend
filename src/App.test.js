import React from 'react';
import Home from './components/Home';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const component = renderer.create(<Home/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});