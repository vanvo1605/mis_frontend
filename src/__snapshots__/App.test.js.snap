// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Home page
  </h1>
</div>
`;
